### PR TITLE
Add local testing topic

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds the local testing topic.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/admin-ui-sdk/configuration.md [hunk 1]
```diff
@@ -1,6 +1,141 @@
 ---
 title: Admin configuration and testing
-description: 
+description: Learn how to configure the Admin to enable local testing of your Admin customizations.
 ---
 
 # Admin configuration and testing
+
+The Adobe Commerce Admin UI SDK enables you to use a local server to view and test your Admin customizations before you submit your app to the Adobe Marketplace.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/configuration.md:8]:
```suggestion
The Adobe Commerce Admin UI SDK allows you to use a local server to view and test your Admin customizations before you submit your app to the Adobe Marketplace.
```

File: src/pages/admin-ui-sdk/configuration.md [hunk 2]
```diff
@@ -1,6 +1,141 @@
...
+
+## Configure the Admin
+
+Navigate to **Stores** > Settings > **Configuration** > **Adobe Services** > **Admin UI SDK** and edit the **General Configuration** screen. When you enable the local service, all calls are automatically redirected to the local server, instead of connecting to Adobe's App Registry. The values you specify must match the contents of your local `server.js` file.
+
+[Test with a sample app](#test-with-a-sample-app) lists a sample `server.js` file.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/configuration.md:14]:
```suggestion
[Test with a sample app](#test-with-a-sample-app) provides a sample `server.js` file.
```

**Review by jhadobe — APPROVED** [review]

File: src/pages/admin-ui-sdk/configuration.md [hunk 3]
```diff
@@ -1,6 +1,141 @@
...
+
+1.  Make sure you have access to the localhost server configuration by entering the following URL in your browser:
+
+   `https://localhost:9090/config`
+
+   The browser displays a JSON file similar to the following:
+
+   ```json
+   {
+    "baseUrl":"https://localhost.adobe.io:9090/",
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/configuration.md:120]: This `baseUrl` should be in line with the one above. I suggest to update to `https://localhost:9090/config`

Comment (asalloum5) [inline src/pages/admin-ui-sdk/configuration.md:120]:
```suggestion
    "baseUrl":"https://localhost:9090/",
```

**Review by asalloum5 — APPROVED** [review]

**Pull request merged**